docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '2'
+services:
+    mongodb:
+        extends:
+            file: docker-compose.yml
+            service: mongodb
+        mem_limit: 4g
+        labels:
+            - triton.cns.services=mongodb
+        network_mode: bridge
```

Comment:
Yeah, I'm not sure on the name of the file, but the readme points out which to use on Triton and which to use elsewhere.

As for the v2 syntax, we need it so that multi-host swarm can deploy the containers to more than one host.  Without it we only have linking and are stuck to just one host in the swarm.  Compose will automatically create a `docker network` for the set of containers that we deploy, which is why we have to say bridge mode on the Triton file so that it won't try to do that.